### PR TITLE
private/marco/add support for scale rotate animations

### PR DESCRIPTION
**Pull request by mcecchetti**
- **slideshow: anim engine: support for scale and rotate animations**
- **slideshow: avoid to run requestSlideImpl forever**

### CODE REVIEW

File: browser/src/slideshow/engine/SlideAnimations.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ function isAttributeSupported(aNodeInfo: AnimateNodeInfo) {
 		console.error('slideshow: missing attributeName');
 		return false;
 	}
+	// return true;
```
Comment (eszkadev) [inline browser/src/slideshow/engine/SlideAnimations.ts:45]: leftover? please remove later

File: browser/src/slideshow/engine/SlideAnimations.ts [hunk 2]
```diff
@@ -69,6 +71,7 @@ function isTransitionFilterSupported(aNodeInfo: TransitionFilterNodeInfo) {
 }
 
 function isTargetSupported(aNodeInfo: AnimateNodeInfo) {
+	// return true;
```
Comment (eszkadev) [inline browser/src/slideshow/engine/SlideAnimations.ts:74]: leftover? please remove later

File: browser/src/slideshow/engine/SlideAnimations.ts [hunk 3]
```diff
@@ -52,7 +53,8 @@ function isTransformSupported(aNodeInfo: AnimateTransformNodeInfo) {
 		console.error('slideshow: missing transformType');
 		return false;
 	}
-	return AnimatedElement.SupportedProperties.has(
+	// return true;
```
Comment (eszkadev) [inline browser/src/slideshow/engine/SlideAnimations.ts:56]: leftover? please remove later

**Review by eszkadev — APPROVED** [review]
looks ok

File: browser/src/slideshow/engine/SlideAnimations.ts (hunk 1 above)
Comment (mcecchetti) [reply browser/src/slideshow/engine/SlideAnimations.ts:45]: it's just useful when you need to relax constrains for developing support for more animations
surely it has to be removed on the long term

**Comment by mcecchetti** [comment]
> looks ok

thanks for your review

**Pull request merged**